sda-auth/info.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"github.com/kataras/iris/v12"
+	log "github.com/sirupsen/logrus"
+)
+
+type Info struct {
+	ClientID  string `json:"client_id"`
+	OidcURI   string `json:"oidc_uri"`
+	PublicKey string `json:"public_key"`
+	InboxURI  string `json:"inbox_uri"`
+}
+
+func (auth AuthHandler) getInfo(ctx iris.Context) {
+	log.Println("in getInfo")
+
+	if auth.OAuth2Config.ClientID != "" && auth.Config.JwtIssuer != "" && auth.Config.S3Inbox != "" {
+		info := Info{ClientID: auth.OAuth2Config.ClientID, OidcURI: auth.Config.JwtIssuer, PublicKey: "todo.pub", InboxURI: auth.Config.S3Inbox}
```

Comment:
I trust when auth is running in a project the auth.Config.JwtIssuer will be the correct one configured to SDA or Big Picture?

For the case of the public key to use though, - according to cli doc - it looks like we need to distinguish between providing a URL where the cli can get either the big picture or the SDA pub key. Should this be a hardcoded condition or how can we determine that from the deployment configuration somehow?

Reply:
The pipeline will know which is the actual key used. 

So either this could be a base64 representation of the archive public C4GH key or another endpoint added to serve the key (would be some additional work to figure out the actual absolute URL to provide but should not be that much).

Reply:
Or the public key could be read from file.

Reply:
So let's say we go with reading the key from the vault and we serve it in /info

I see in both these below we get c4gh.pub.pem:
https://github.com/NBISweden/LocalEGA-SE-Deployment/blob/master/deployment-templates/standalone/sda-standalone-all-neic.tf#L783
https://github.com/NBISweden/BigPicture-Deployment/blob/c339ba9d5e6bd14b8dbf3729685206ae3e20297c/sc_cluster/services/bigpicture/bp-prod.tf#L1113
then also here?: 
https://github.com/NBISweden/LocalEGA-SE-Deployment/blob/c27c3f482f67eb794274d89c17709c1686729510/dev-tools/helmfiles/svc-values.yaml#L29

I'm trying to figure out the changes needed and where could I expect to "see" the file through sda-auth.
Can I just assume that I will be able to read from a file in `/c4gh.pub.pem` ?

Also if there's an easy way to test this once I do the changes.

Reply:
I can't see those, but for this code, you will either see the key as a file in the file system (in which case it should be configurable where). Alternatively, it can possibly be accepted as an environment variable.

My recommendation is to add a config for where the C4GH public key should be read from.